File: baekjoon/p01919/Main.java
package baekjoon.p01919;

import java.util.Scanner;

public class Main {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.println(solve(sc.next(), sc.next()));
    }

    private static int solve(String a, String b) {
        final int size = 26;
        int[] cntA = new int[size];
        int[] cntB = new int[size];

        for (char ch : a.toCharArray()) {
            cntA[ch - 'a']++;
        }

        for (char ch : b.toCharArray()) {
            cntB[ch - 'a']++;
        }

        int ret = 0;
        for (int i = 0; i < size; i++) {
            ret += Math.abs(cntA[i] - cntB[i]);
        }

        return ret;
    }

}

File: baekjoon/p01919/Main.kt
package baekjoon.p01919

import kotlin.math.abs

fun main() {
    println(solve(readLine()!!, readLine()!!))
}

private fun solve(a: String, b: String): Int {
    val size = 26
    val cntA = IntArray(size)
    val cntB = IntArray(size)

    for (i in 0 until a.length) {
        val idx = a[i] - 'a'
        cntA[idx]++
    }

    for (i in 0 until b.length) {
        val idx = b[i] - 'a'
        cntB[idx]++
    }

    return (0 until size).sumBy { abs(cntA[it] - cntB[it]) }
}